CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# CHANGELOG for `contextual_logger`
+
+Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.5.1] - 2019-03-10 [diff](https://github.com/Invoca/contextual_logger/compare/v0.5.0...0.5.1)
```

Comment:
🤔 do you mean 2020?

Reply:
Haha, yes! I think I copied their example a bit too closely. :) Fixing...